lib/api/util/index.js

```diff
@@ -0,0 +1,5 @@
+'use strict';
+
+const fileExport = require('./file-export');
+
+module.exports = Object.assign({}, fileExport);
```

Comment:
@allanchau, why the `Object.assign` here?

Reply:
Just planning ahead for when we add more functions here. We don't need it now though, did you want me to remove it?

Reply:
@allanchau, of course, yeah, that's better and more inline with our standard coding...